### DISCUSSION TITLE
Setting height to "auto" with a timeline causes wavesurf to expand indefinitely

### DISCUSSION THREAD

**Discussion by rodsmyname** [katspaugh/wavesurfer.js#3722] [General]
This looks like a bug, but any advice would be great.

Simply setting height to "auto" during initialization and having the timeline plugin (at the bottom) causes wavesurfer to expand vertically indefinitely. This can be seen in the Timeline plugin example:
<img width="1108" alt="Screenshot 2024-05-29 224143" src="https://github.com/katspaugh/wavesurfer.js/assets/111706981/822f5163-d413-45a3-a459-c3812ae13576">

**Comment by katspaugh** [comment]
It’s a bug indeed.
